ext/node/polyfills/internal/buffer.mjs

```diff
@@ -2598,6 +2602,40 @@ export function isAscii(input) {
   ], input);
 }
 
+export function transcode(source, fromEnco, toEnco) {
+  if (!isUint8Array(source)) {
+    throw new codes.ERR_INVALID_ARG_TYPE(
+      "source",
+      ["Buffer", "Uint8Array"],
+      source,
+    );
+  }
+  if (source.length === 0) {
+    return Buffer.alloc(0);
+  }
+  fromEnco = fromEnco === "binary" ? "latin1" : fromEnco;
+  toEnco = toEnco === "binary" ? "latin1" : toEnco;
```

Comment:
We may want to call [`normalizeEncoding`](https://github.com/denoland/deno/blob/3881b7173445ab7f68ec94d5dedbb1cb1c1978ec/ext/node/polyfills/_utils.ts#L101-L106) here. It takes care of casing.